packages/ag-charts-community/src/chart/series/hierarchy/hierarchySeries.ts

```diff
@@ -69,7 +82,10 @@ export class HierarchyNode implements SeriesNodeDatum {
     }
 }
 
-export abstract class HierarchySeries<S extends SeriesNodeDatum> extends Series<S> {
+export abstract class HierarchySeries<
+    TNode extends Node = Group,
+    TDatum extends SeriesNodeDatum = any,
```

Comment:
Could we default to `SeriesNodeDatum` rather than `any` (as this is promiscuous by default)?